plugin.go

```diff
@@ -307,8 +308,12 @@ func packageVersionsCommand(name string) *exec.Cmd {
 }
 
 // publishCommand runs the publish command
-func publishCommand() *exec.Cmd {
-	return exec.Command("npm", "publish")
+func publishCommand(config Config) *exec.Cmd {
+	if len(config.Tag) == 0 {
```

Comment:
How about we turn this into a list of string and then if there's a tag we append those two to it?

Also would prefer it just being a string passed in unless you also want to add support for `--access` while you're here. https://docs.npmjs.com/cli/publish